### PR TITLE
sherlock: add page

### PR DESCRIPTION
**Pull request by Hephaestus14089**
add page for sherlock

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

Comes from #7606

### CODE REVIEW

**Comment by navarroaxel** [comment]
Hi @Hephaestus14089, welcome. you don't need to create a new PR to attack the review of a collaborator. Just add the changes here via new commits, otherwise it's harder to us to track suggestions and changes.

**Comment by CleanMachine1** [comment]
Please move the display help command to the bottom, and add a more information link, such as the Sherlock GitHub

**Comment by CleanMachine1** [comment]
What was the problem with the other PR?

File: pages/linux/sherlock.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# sherlock
+
+> Find usernames across social networks.
```
Comment (navarroaxel) [inline pages/linux/sherlock.md:3]:
```suggestion
> Find usernames across social networks.
> More information: <https://github.com/sergius02/Sherlock>.
```

File: pages/linux/sherlock.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`sherlock --help`
+
+- Save result to file (if using single username):
+
+`sherlock {{username}} --output {{filename}}`
+
+- Save result to folder (if using multiple usernames):
+
+`sherlock {{[usernames]}} --folderoutput {{foldername}}`
```
Comment (navarroaxel) [inline pages/linux/sherlock.md:15]: We don't use `[]` in examples
```suggestion
`sherlock {{usernames}} --folderoutput {{foldername}}`
```

File: pages/linux/sherlock.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Make requests over Tor:
+
+`sherlock --tor {{username}}`
+
+- Make requests over Tor with new Tor circuit after each request:
```
Comment (navarroaxel) [inline pages/linux/sherlock.md:21]:
```suggestion
- Make requests over Tor with a new Tor circuit after each request:
```

File: pages/linux/sherlock.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+`sherlock --unique-tor {{username}}`
+
+- Make requests over a proxy:
+
+`sherlock {{username}} --proxy {{proxyurl}}`
+
+- Browse to all results on default browser:
+
+`sherlock {{username}} --browse`
+
```
Comment (navarroaxel) [inline pages/linux/sherlock.md:32]:
```suggestion
```

File: pages/linux/sherlock.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+- Make requests over Tor with new Tor circuit after each request:
+
+`sherlock --unique-tor {{username}}`
+
+- Make requests over a proxy:
+
+`sherlock {{username}} --proxy {{proxyurl}}`
+
+- Browse to all results on default browser:
```
Comment (navarroaxel) [inline pages/linux/sherlock.md:29]: This example open a report? I don't get it.

File: pages/linux/sherlock.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
+# sherlock
+
+> Find usernames across social networks.
+
+- Display help:
+
+`sherlock --help`
+
+- Save result to file (if using single username):
```
Comment (navarroaxel) [inline pages/linux/sherlock.md:9]: the result of what?

File: pages/linux/sherlock.md (hunk 5 above)
Comment (Hephaestus14089) [reply pages/linux/sherlock.md:26]: It finds the username in the social networks and opens the link of the profiles.

File: pages/linux/sherlock.md (hunk 6 above)
Comment (Hephaestus14089) [reply pages/linux/sherlock.md:6]: Stores the links of the profiles with the provided username/usernames.

Comment (navarroaxel) [reply pages/linux/sherlock.md:6]: Wait, `sherlock` opens the an UI app? a GTK window, I mean.

Comment (Hephaestus14089) [reply pages/linux/sherlock.md:6]: Sherlock is a cli tool. With the --browse option, when it is finished with the searching, it opens the links to the profiles it has found in the default browser.

Comment (navarroaxel) [reply pages/linux/sherlock.md:6]: Ok, perhaps is better:
```suggestion
- Search for a specific username on social networks saving the results to a file:
```

File: pages/linux/sherlock.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+`sherlock --tor {{username}}`
+
+- Make requests over Tor with a new Tor circuit after each request:
+
+`sherlock --unique-tor {{username}}`
+
+- Make requests over a proxy:
+
+`sherlock {{username}} --proxy {{proxyurl}}`
```
Comment (navarroaxel) [inline pages/linux/sherlock.md:24]: `uri` is the correct term
```suggestion
`sherlock {{username}} --proxy {{proxy_uri}}`
```

File: pages/linux/sherlock.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
+# sherlock
+
+> Find usernames across social networks.
+> More information: <https://github.com/sherlock-project/sherlock>.
+
+- Save result to file (if using single username):
+
+`sherlock {{username}} --output {{filename}}`
```
Comment (navarroaxel) [inline pages/linux/sherlock.md:8]: We use this placeholders for files:
```suggestion
`sherlock {{username}} --output {{path/to/file}}`
```

File: pages/linux/sherlock.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+> Find usernames across social networks.
+> More information: <https://github.com/sherlock-project/sherlock>.
+
+- Search for a specific username on social networks saving the results to a file:
+
+`sherlock {{username}} --output {{path/to/file}}`
+
+- Save result to folder (if using multiple usernames):
+
+`sherlock {{usernames}} --folderoutput {{path/to/directory}}`
```
Comment (navarroaxel) [inline pages/linux/sherlock.md:12]:
```suggestion
`sherlock {{username1 username2 ...}} --folderoutput {{path/to/directory}}`
```

File: pages/linux/sherlock.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
+# sherlock
+
+> Find usernames across social networks.
+> More information: <https://github.com/sherlock-project/sherlock>.
+
+- Search for a specific username on social networks saving the results to a file:
+
+`sherlock {{username}} --output {{path/to/file}}`
+
+- Save result to folder (if using multiple usernames):
```
Comment (navarroaxel) [inline pages/linux/sherlock.md:10]:
```suggestion
- Search for specific usernames on social networks saving the results into a directory:
```

File: pages/linux/sherlock.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+`sherlock {{usernames}} --folderoutput {{path/to/directory}}`
+
+- Make requests over Tor:
```
Comment (navarroaxel) [inline pages/linux/sherlock.md:14]: Perhaps is better
```suggestion
- Search for a specific username on social networks using the Tor network:
```

File: pages/linux/sherlock.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+
+`sherlock --tor {{username}}`
+
+- Make requests over Tor with a new Tor circuit after each request:
+
+`sherlock --unique-tor {{username}}`
+
+- Make requests over a proxy:
```
Comment (navarroaxel) [inline pages/linux/sherlock.md:22]:
```suggestion
- Search for a specific username on social networks using a proxy:
```

File: pages/linux/sherlock.md (hunk 5 above)
Comment (navarroaxel) [reply pages/linux/sherlock.md:26]: Perhaps is better
```suggestion
- Search for a specific username on social networks and open results in the default web browser:
```

**Review by navarroaxel — APPROVED** [review]

**Review by 258204 — APPROVED** [review]
Thanks for the new page, looks good!

File: pages/linux/sherlock.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+
+`sherlock --tor {{username}}`
+
+- Make requests over Tor with a new Tor circuit after each request:
+
+`sherlock --unique-tor {{username}}`
+
+- Search for a specific username on social networks using a proxy:
+
+`sherlock {{username}} --proxy {{proxy_uri}}`
```
Comment (CleanMachine1) [inline pages/linux/sherlock.md:24]:
```suggestion
`sherlock {{username}} --proxy {{proxy_url}}`
```

**Pull request merged**